src/plugins/explore/public/components/experience_banners/classic_experience_banner/classic_experience_banner.tsx

```diff
@@ -24,7 +24,7 @@ export const ClassicExperienceBanner = ({ navigateToExplore }: ClassicExperience
       data-test-subj="exploreClassicExperienceBanner"
       size="s"
```

Comment:
I could've removed this `size="s"` prop from the Callout component, but then that makes the banner have extremely large paddings. So i opted for the `EuiText` approach